### PR TITLE
Multilingual map.legend_at_web

### PR DESCRIPTION
**Pull request by pfirpfel**
Implements #704

TODO:

- [x] add more tests:
  - [x] test renderer by requesting unavailable lang, non-default lang
-  [x] add documentation
-  [x] add ~db migration script~ *migration* documentation
-  [x] check print templates
-  [x] make the xml rendering work and add a test for it

### CODE REVIEW

**Comment by jwkaltz** [comment]
Because this will involve a database change, it should not be included in release 1.3.1. Instead we should release 1.3.1 just before this PR is ready, then start a 1.4 version and finish this PR.

**Comment by pfirpfel** [comment]
Move tasks of adding more tests for XML rendering to #779

**Comment by pfirpfel** [comment]
@peterschaer Could you test the modified field? The JSON extract and the mapfish print PDF should be fine from my testing.

**Comment by pfirpfel** [comment]
Cherry-picked #782 and fixed XML rendering

File: tests/records/test_extract.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def test_init():
         'test_link',
         1,
         1.0,
-        'test_legend'
+        {'lang': 'test_legend'}
```
Comment (ger-benjamin) [inline tests/records/test_extract.py]: It will not fail with `lang` as lang ?

File: tests/webservice/test_getegrid.py [hunk 2]
```diff
@@ -160,7 +160,7 @@ def test_get_egrid_response():
         view_service = ViewServiceRecord('test',
                                          1,
                                          1.0,
-                                         'test',
+                                         {'lang': 'test'},
```
Comment (ger-benjamin) [inline tests/webservice/test_getegrid.py]: same as upper

File: tests/records/test_extract.py (hunk 1 above)
Comment (pfirpfel) [reply tests/records/test_extract.py]: No, it's not getting checked here. But I agree, it is confusing. I change it.

**Pull request merged**